docker2singularity.sh

```diff
@@ -270,8 +270,11 @@ chmod +x $build_sandbox/.singularity.d/runscript;
 echo "(5/10) Setting ENV variables..."
 
 # First add templates for actions and env
-cp -R /scripts/env $build_sandbox/.singularity.d/
-cp -R /scripts/actions $build_sandbox/.singularity.d/
+## Removed to avoid permissions issue with scripts (see #66)
+## cp -R /scripts/env $build_sandbox/.singularity.d/
+## cp -R /scripts/actions $build_sandbox/.singularity.d/
+# Create env scripts directory
+mkdir $build_sandbox/.singularity.d/env; chmod 755 $build_sandbox/.singularity.d/env
```

Comment:
Could we:
 - do mkdir -p so the command won't fail if it (for some future reason) exists,
 - move each command to a single line
 - also in case the user puts other content there, perhaps chmod -R ?
 -  no need to comment out the original lines, delete is good (and if you want to include the comment that adding specific files was removed to point to the issue that works!)

In that we don't need those scripts/env and scripts/actions anymore, does it make sense to remove them from the repository?

Reply:
One query on the `chmod -R` - if other files are being placed into the directory by the user, this would make them all executable. If the user is putting additional content into the `env` directory, shouldn't they be responsible for ensuring the correct permissions?

Other changes made. Happy to include the `-R` if you think this is a better approach. Let me know if you'd like any other mods.

Re the env and actions scripts - wasn't clear as yet where these are actually coming from now that they're not being copied in - I guess if they're not being used at all then they could be removed but maybe it's worth leaving them there for reference for now?

Reply:
I'm fairly indifferent about the -R actually, I wanted to get your feedback on it, it's probably best to leave as is until we have evidence that it would be needed :P 

It wouldn't hurt to leave, I agree it's nice to see them there since it's generally hard to find them in the Singularity repository. In this case, could you add a comment to the README that mentions they aren't used, but just for reference?

Reply:
After that, LGTM!

Reply:
Ok, great. I'll update the README. Do you have any thoughts on the best place in the README to mention the scripts? It's not really something for the FAQ section or the Development section. Could just add an additional heading before Acknowledgments, "Singularity scripts" or similar.

Reply:
Sure! How about in the "Legacy" section, after this bit:

> Intermediate versions built on Docker Hub. A tag with prefix v corresponds to a release of the Singularity software, while the others are in reference to releases of Docker.

We can update to be something like:

> Intermediate versions built on Docker Hub. A tag with prefix v corresponds to a release of the Singularity software, while the others are in reference to releases of Docker. Previously used [scripts](https://github.com/singularityware/docker2singularity/tree/master/scripts), including environment and action files, are provided for reference.

Reply:
Thanks, LGTM, README updated with a very minor tweak to the text.